includes/class-wc-webhook.php

```diff
@@ -306,6 +307,31 @@ protected function is_already_processed( $arg ) {
 		return false !== array_search( $arg, $this->processed, true );
 	}
 
+	/**
+	 * Make retry enabled determination.
+	 *
+	 * @since 4.4.0
+	 * @param int $arg Resource ID
+	 * @param WP_Error|HTTP_Response $response
+	 * @return bool
+	 */
+	public function is_retryable( $arg, $response ) {
+		$webhook_retry_enabled = $this->get_retry_enabled();
+
+		// If we reached the limit of failures we won't reschedule. (Why doesn't status update, bug?)
+		$failed = ( $this->get_failure_count() > apply_filters( 'woocommerce_max_webhook_delivery_failures', 5 ) );
```

Comment:
I wonder if we should be using the `failure_count` for retries? Right now failures are used to automatically disable webhooks that aren't working correctly which seems kind of counter to the heart of this change. I almost feel like there should be a separate `retry_count` and when retries are enabled it won't trigger a `failure` until it reaches the retry count?

Reply:
I didn't implement this yet -- I'll play with this next.